support-frontend/assets/pages/supporter-plus-landing/components/threeTierCard.tsx

```diff
@@ -190,7 +191,7 @@ const discountSummaryCopy = (
 		paymentFrequency === 'ANNUAL' ? durationMonths / 12 : durationMonths;
 
 	return `${formattedPromotionPrice}/${period} for ${
-		period === 'year' ? 'the first' : ''
+		period === 'year' || duration === 1 ? 'the first' : ''
```

Comment:
Interesting how we always say `the first` for years regardless of the period, e.g. `the first year`, `the first 2 years` (maybe we never have multi year promos). But for months we want `the first month` but not `the first 2 months` (preferring simply `2 months`).

Reply:
Yes, there was a 12 month test in the build for multiple months asking for `the first` to be removed but anythong over a month the offer line would be too long anyway. As for yearly, I presume a promo needs to run for a year minimum would guess (whether thats enforced I suspect not though).